docs/plugins/dig.rst

```diff
@@ -143,9 +143,10 @@ Designation options:
 ``clear``
     Clear any designations.
 
-You can also pass a ``-z`` option, which restricts designations to the current
-z-level and down. This is useful when you don't want to designate tiles on the
-same z-levels as your carefully dug fort above.
+You can also pass a ``-z`` and/or a ``+z``` option, which restricts designations to the
```

Comment:
clever, but not in line with how DFHack options generally work. 

How about if we align `-z` with the definition used by `dig-now` https://docs.dfhack.org/en/stable/docs/tools/dig-now.html?highlight=dig-now
and add --zdown and --zup to mean "this z level and down" and "this z level and up"

Reply:
I've implemented these changes, and added single-character equivalent options.

`--zdown` == `-d`, `--zup` == `-u`, `--cur-zlevel` == `-z`.